engine/runtime/src/test/java/org/enso/interpreter/test/SharedEngineTest.java

```diff
@@ -0,0 +1,63 @@
+package org.enso.interpreter.test;
+
+import java.io.ByteArrayOutputStream;
+import java.nio.file.Paths;
+
+import org.enso.polyglot.RuntimeOptions;
+import org.graalvm.polyglot.Context;
+import org.graalvm.polyglot.Engine;
+import org.graalvm.polyglot.Source;
+import static org.junit.Assert.assertEquals;
+import org.junit.Before;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+public class SharedEngineTest extends TestBase {
```

Comment:
This is a test taken from branch [wip/jtulach/NoTypeInNodePlease_6809](https://github.com/enso-org/enso/commit/567f80218c66ae97154b1468478393a2b511ba91). The test seems to pass OK with `EXCLUSIVE` context.